apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx

```diff
@@ -229,6 +229,12 @@ const containWordsInSquareBrackets = (str) => {
     return words.length !== 0
 }
 
+const getTitleTemplateChanged = (form) => {
+    const { title } = form.getFieldsValue(['title'])
+    // NOTE: this check blocks any sending of [] in the news body
```

Comment:
```suggestion
    // NOTE: this check blocks any sending of [] in the news title
```

Reply:
we can also just remove this comment. 235 is obvious enough and does not need a comment